packages/api/src/webhooks/cannon.ts

```diff
@@ -200,11 +218,22 @@ export default class WebhookCannon {
     return newInterval | 0;
   };
 
-  retry(trigger: messages.WebhookTrigger) {
+  retry(
+    trigger: messages.WebhookTrigger,
```

Comment:
Ops, you might need to do a manual `rebase` of your branch with master, cause the changes of the first PR are showing here. LMK if you need help with it. This command should do magic: `git rebase --onto master <last commit on this PR history that is not from this PR> <this PR branch>`